BUILD.gn

```diff
@@ -591,6 +591,7 @@ if (is_mac) {
 
     libs = [
       "AVFoundation.framework",
+      "LocalAuthentication.framework",
```

Comment:
_(nit)_ is the order important, or can this new lib be added in alphabetical order like the others?